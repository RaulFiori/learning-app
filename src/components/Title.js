import React from 'react';
import PropTypes from 'prop-types';
import { Typography } from '@material-ui/core';

const Title = props => {
  const { children, ...other } = props;
  return (
    <Typography {...other} variant="h6">
      {children}
    </Typography>
  );
};

Title.propTypes = {
  children: PropTypes.string.isRequired
};

export default Title;
