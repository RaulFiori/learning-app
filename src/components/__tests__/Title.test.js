import React from 'react';

import renderer from 'react-test-renderer';
import Title from '../Title';

it('renders correctly', () => {
  const tree = renderer.create(<Title>ol√°</Title>).toJSON();
  expect(tree).toMatchSnapshot();
});
