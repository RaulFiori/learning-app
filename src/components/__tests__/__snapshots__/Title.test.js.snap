// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h6
  className="MuiTypography-root MuiTypography-h6"
>
  ol√°
</h6>
`;
